client/formSchemas/signUpSchema.ts

```diff
@@ -0,0 +1,22 @@
+import { z } from 'zod';
+import { SignUpRequest as OpenApiSignUpRequest } from 'openapi-generator/api';
+
+export interface SignUpRequest extends OpenApiSignUpRequest {
+  password_confirmation: string;
+}
```

Comment:
[IMO]
interfaceは勝手に拡張される可能性あるからあんまり使いたくないかな。

```suggestion
type SignUpRequest = {
  password_confirmation: string;
} & OpenApiSignUpRequest;
```

でいけそう

https://zenn.dev/luvmini511/articles/6c6f69481c2d17

Reply:
typeとinterfaceで`&`使えるのか